### PR TITLE
LOG-1380: Updating alerts to use runbook_url and have different base URIs

### PR DESCRIPTION
**Pull request by ewolinetz**
### Description
For upstream and downstream, we use different URLs for our playbooks.

This will need to be cherrypicked to 5.1 as well

/cc @periklis @lukas-vlcek

### Links
- Related to PR(s): https://github.com/openshift/openshift-docs/pull/32586
- JIRA: https://issues.redhat.com/browse/RHDEVDOCS-2791

### CODE REVIEW

**Comment by ewolinetz** [comment]
cc @sichvoge @rolfedh

File: Dockerfile.dev [hunk 1]
```diff
@@ -1,3 +1,4 @@
+{'from': [{'source': 'openshift-golang-builder\\:v(?:[\\.0-9\\-]*).*', 'target': 'registry.ci.openshift.org/openshift/release:golang-1.14 AS builder'}, {'source': 'openshift-ose-base\\:v(?:[\\.0-9\\-]*)', 'target': 'docker.io/centos:8 AS centos'}], 'env': [{'source': 'RUNBOOK_BASE_URL=.*', 'target': 'RUNBOOK_BASE_URL="https://github.com/openshift/elasticsearch-operator/blob/master/docs/alerts.md"'}]}
```
Comment (periklis) [inline Dockerfile.dev:1]: Is this block intended or a generation mistake?

Comment (ewolinetz) [reply Dockerfile.dev:1]: good catch, i think this is a generation issue and why the images aren't building

Comment (ewolinetz) [reply Dockerfile.dev]: i left a debugging print statement in the generate script was the issue...

**Review by jcantrill — COMMENTED** [review]
/lgtm

**Pull request merged**